Sources/Get/Fetchable.swift

```diff
@@ -27,4 +27,7 @@ protocol Fetchable {
     // so Package has a version { get } already, we cannot add
     // a set, so instead we have to have this protocol func
     func setVersion(_ newValue: Version) throws
+
+    /// The localpath whenre Fetchable is stored
+    var localPath: String { get }
```

Comment:
`local path`, and comments should use English punctuation (trailing period, in this case).

Why not just use `path`, since that matches what `Package` already has?